judge/migrations/0058_problem_currator_field.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Kirito Feng on shrooms
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0057_blue_pretests'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='problem',
+            name='curators',
+            field= models.ManyToManyField(Profile, verbose_name=_('curators'), blank=True, related_name='curated_problems',
```

Comment:
Did you seriously write this yourself? You should probably generate it, but look at that `_`.